### ISSUE TITLE
Error: Class "humhub\compat\CActiveForm" not found

### ISSUE THREAD

**Issue by yurabakhtin** [humhub-contrib/breakingnews#16]
Configuration page doesn't work on HumHub version 1.11+

![deprecated_form](https://user-images.githubusercontent.com/6995524/159877195-daffa334-2f27-4613-8956-d130b23bcd9b.png)

**Comment by yurabakhtin** [comment]
On attaching image to news content:
https://community.humhub.com/content/perma?id=253478
![error_attach_file](https://user-images.githubusercontent.com/6995524/159881914-c4d1c619-9310-41da-af45-5a579e5ac7f3.png)

**Comment by yurabakhtin** [comment]
@luke- PR https://github.com/humhub-contrib/breakingnews/pull/17:

I have replaced the deprecated class `CActiveForm`.

Also when a file is attached we should fill the columns `object_model` and `object_id` in the table `file`, but model should be instance of `humhub\components\ActiveRecord` because we have such restriction from RichText class side.
So I have created a new class for this `humhub\modules\breakingnews\models\Setting`, probably it is not good solution but I cannot find another. I remember we already had the same issue somewhere, but I cannot find where, it was in some setting field too, and if I am not mistaken we used there the core class `humhub\models\Setting`, but I could use it here today, because it is extended from `yii\db\ActiveRecord` instead of expected `humhub\components\ActiveRecord` on RichText attaching side.

I have detected the same issue in the module `termsbox`, I will do a fix there like this.
